prometheus-alertmanager.yaml

```diff
@@ -45,6 +44,35 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+    description: "compat package with bitnami/alertmanager image"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/alertmanager/bin/
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/alertmanager/conf/
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/alertmanager/data/
+          mkdir -p ${{targets.contextdir}}/etc/alertmanager
+          mkdir -p ${{targets.contextdir}}/alertmanager
+          chmod g+rwX ${{targets.contextdir}}/opt/bitnami/alertmanager/data/
+          ln -sf /usr/bin/alertmanager ${{targets.contextdir}}/opt/bitnami/alertmanager/bin/alertmanager
+          ln -sf /usr/bin/amtool ${{targets.contextdir}}/opt/bitnami/alertmanager/bin/amtool
+          ln -sf /etc/alertmanager/alertmanager.yml ${{targets.contextdir}}/opt/bitnami/alertmanager/conf/config.yml
+          ln -sf /opt/bitnami/alertmanager/conf ${{targets.contextdir}}/etc/alertmanager
+          ln -sf /opt/bitnami/alertmanager/data ${{targets.contextdir}}/alertmanager
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+      pipeline:
+        - runs: |
+            /opt/bitnami/alertmanager/bin/alertmanager --version | grep ${{package.version}}
+            /opt/bitnami/alertmanager/bin/alertmanager --help
+            /opt/bitnami/alertmanager/bin/amtool --version | grep ${{package.version}}
+            /opt/bitnami/alertmanager/bin/amtool --help
+
```

Comment:
Can we have a `test/daemon-check-output` pipeline here to spin up with defined env vars and its entrypoint/cmd?

Reply:
Then simple `curl` test for `/api/v2/alerts` endpoint? Or status / health endpoints.